Tests/NestTests/Arguments/InstallTargetTests.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import Testing
+@testable import nest
+@testable import NestKit
+
+struct InstallTargetTests {
+    @Test
+    func testInstallTarget() throws {
+        let installTarget = InstallTarget(argument: "artifactBundle.zip")
+        switch installTarget {
+        case .git(let gitURL):
+            fatalError()
+        case .artifactBundle(let url):
+            #expect(url == URL(string: "artifactBundle.zip"))
+        case nil:
+            fatalError()
+        }
```

Comment:
`Issue.record` is better than `fatalError`, because it doesn't stop the test process.
```suggestion
        let installTarget = InstallTarget(argument: "artifactBundle.zip")
        guard case .artifactBundle(let url) = installTarget else {
            Issue.record("installTarget needs to be artifactBundle.")
            return
        }
        #expect(url == URL(string: "artifactBundle.zip"))
```